README.md

```diff
@@ -144,11 +144,11 @@ var config = {
 var authClient = new OktaAuth(config);
 ```
 
-The `tokenManager` will **automatically refresh tokens** for you when they expire. To disable this feature, set `autoRefresh` to false.
+The `tokenManager` will **automatically renew tokens** for you when they expire. To disable this feature, set `autoRenew` to false.
```

Comment:
👍 

Why are we bolding this though? I think it can be normal text.

Reply:
We also need to make it clear that this is done on-demand, and not on a timer.  I don't think that's obvious yet.